MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/analytics/AnalyticsTracker.java

```diff
@@ -0,0 +1,134 @@
+package com.mapbox.mapboxandroiddemo.analytics;
+
+
+import android.os.Build;
+import android.support.annotation.NonNull;
+
+import com.segment.analytics.Analytics;
+import com.segment.analytics.messages.IdentifyMessage;
+import com.segment.analytics.messages.ScreenMessage;
+import com.segment.analytics.messages.TrackMessage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by langstonsmith on 4/25/17.
+ */
+
+public class AnalyticsTracker {
+
+  private static final AnalyticsTracker INSTANCE = new AnalyticsTracker();
+
+  private String openedApp = "Opened app";
+  private String clickedOnSignInButtonEventName = "Clicked on sign in button";
+  private String clickedOnCreateAccountButtonEventName = "Clicked on create account button";
+  private String clickedOnNavDrawerSectionEventName = "Clicked on nav drawer section";
+  private String clickedOnIndividualExampleEventName = "Clicked on individual example";
+  private String sectionNameMapKey = "section name";
+  private String exampleNameMapKey = "example name";
+  private String isTabletMapValue = "tablet";
+  private String isPhoneMapValue = "phone";
+
+
+  public static AnalyticsTracker get() {
```

Comment:
- for convention we use `getInstance()` at Mapbox for getting a Singleton.
 - you aren't overriding the constructor of the class with a private constructor, users are able to make multiple instances of this class with `new AnalyticsTracker()`.
 - this is not thread safe